qa/smoke-test-multinode/src/test/resources/rest-api-spec/test/smoke_test_multinode/20_tsdb_consistency.yml

```diff
@@ -0,0 +1,103 @@
+# Test the mode:time_series properly groups by _tsid. If we could put this in
+# rest-api-spec we would, but it requires painless.
+
```

Comment:
Should we skip setup here as well? I am curious why this didn't fail in bwc test?

Reply:
Oh! It's because we don't run this against a mixed version cluster! Huh. I'll have a think about how to do that.

Reply:
I think tests were also disabled, because it now fails in https://github.com/elastic/elasticsearch/pull/77731/files#diff-ee59a8405494ffd00e1b28d5f3ed8d537b51fa846deed022fac7fc42328fc698R13